src/OpenLoco/Map/TileManager.cpp

```diff
@@ -26,7 +26,7 @@ namespace OpenLoco::Map::TileManager
         tile_coord_t tileX = x / 32;
         tile_coord_t tileY = y / 32;
 
-        size_t index = ((y << 9) | x) >> 5;
+        size_t index = (tileY << 9) | tileX;
```

Comment:
Why was the `>> 5` dropped here?

Reply:
So I've switched it to tile coords as they are already computed just before this. Tile coords are 32x smaller (>>5). The reason i made the change was i wanted to mask off the additional bits of a full coord.